packages/ui/src/components/text.tsx

```diff
@@ -5,7 +5,7 @@ import { cva } from 'class-variance-authority'
 
 import { cn } from '../utils/cn'
 
-const textVariants = cva('text-base', {
+const textVariants = cva('text-base tracking-tight', {
```

Comment:
We already have the tracking-tight class applied to the body, so there's no need to include it here.